lib/puppet/provider/keystore_certificate/openssl.rb

```diff
@@ -53,7 +53,8 @@ def add_certificate
         '-deststorepass:file', resource[:password_file],
         '-srcalias', resource[:alias],
         '-destalias', resource[:alias],
-        '-J-Dcom.redhat.fips=false'
+        '-J-Dcom.redhat.fips=false',
+        '-keyalg', 'RSA'
```

Comment:
`-importkeystore` doesn't have `-keyalg`

Reply:
Fair, it did not error out but it is not needed.